app/Http/Controllers/admin/AdminController.php

```diff
@@ -347,7 +347,8 @@ public function email_group(Request $request)
         // For each result, construct and send the email
         $tokens = [];
         foreach ($query_result as $row) {
-            $send_to = $row->email;
+            // Some emails are stored under a different column, and we get them "as Email"
+            $send_to = isset($row->email) ? $row->email : $row->Email;
```

Comment:
Just in terms of readability, it seems a little confusing to have to remember that `email` and `Email` are different. Could you grab them as something more specific like `nominee_email` or something?

Reply:
@sbsmith86 I thought about that too, but some of them are the emails of nominees and some of them are the emails of nominators. I guess that could all shorten to `nom_email`?